index.js

```diff
@@ -19,8 +21,8 @@ var isActualNaN = function (value) {
 };
 var NON_HOST_TYPES = {
   'boolean': 1,
-  number: 1,
-  string: 1,
+  number:    1,
```

Comment:
please only put one space after a colon - nothing should be visually aligned in code, only semantically indented.